.github/workflows/ci.yml

```diff
@@ -21,13 +21,13 @@ env:
 jobs:
   cancel_previous_runs:
     name: Cancel Previous Runs
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - uses: styfle/cancel-workflow-action@0.11.0
         with:
           access_token: ${{ secrets.GITHUB_TOKEN }}
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
```

Comment:
Latest is no 24.04, which broke our builds

Reply:
no surprise... :-(

Reply:
TBH the build itself worked, but the docker image environment is a 22.04, and it failed to run there then.